platform_api/orchestrator/kube_client.py

```diff
@@ -1575,25 +1590,27 @@ def are_sufficient(self, pod: PodDescriptor) -> bool:
         return (
             self.cpu_mcores >= r.cpu_mcores
             and self.memory >= r.memory
-            and self.gpu >= (r.gpu or 0)
+            and self.nvidia_gpu >= (r.nvidia_gpu or 0)
```

Comment:
mb we need to ensure amd gpus are sufficient too?

Reply:
yes, thanks